### ISSUE TITLE
add a way for mark links (defined by "href") to open in a new window

### ISSUE THREAD

**Issue by silky** [vega/vega-lite#3795]
given something like:

```
    "mark": { "type": "circle", "size": 100, "cursor": "pointer" },
    "encoding": {
        "href":    {"field": "some-url", "type": "nominal"}
    }
```

when clicked, the url is opened in the existing window. it would be great if it you could specify a new window to be opened.

**Comment by silky** [comment]
no worries; i'm happy with the shift-click; thanks!

**Comment by domoritz** [comment]
Excellent. If you think the docs could be clearer a out this, you can send a PR to update them.

**Comment by irbg** [comment]
+1 This would still be a useful in the vega-lite spec. Is there any update on supporting this in Vega?

**Comment by Casyfill** [comment]
Just want to circle back on that - in my humble, we could just switch the default way to open new tab, without changing the  specs, no?

**Comment by domoritz** [comment]
Yes, that's what's https://github.com/vega/vega-embed/issues/77 was for.

**Comment by kennybui-data-ai** [comment]
@domoritz  can you provide a link to the documentation that shows how to:

> override the loader

**Comment by domoritz** [comment]
It's in the readme. The loader is one of the properties in the options object.

**Comment by kennybui-data-ai** [comment]
In the chart `spec`, I specified 
```
"usermeta": {
            "embedOptions": {
                'theme': 'dark',
                'loader': {'target': '_blank'}
            }
        }
```
This did not work, but the [readme](https://github.com/vega/vega-embed#options) mentions this should work. The theme is dark as expected though.

I passed `{'loader': {'target':'_blank'}}` to the opt argument in vegaEmbed and this did work as expected. any reason why the `usermeta.embedOptions` did not work?

**Comment by domoritz** [comment]
Strange. I filed https://github.com/vega/vega-embed/issues/625.

**Comment by domoritz** [comment]
Fixed in https://github.com/vega/vega-embed/issues/625. Can you take a look just to double-check my logic?

**Comment by kennybui-data-ai** [comment]
`usermeta.embedOptions.loader` for vega-embed v6.15.1 works as expected now. Thank you @domoritz ❤️
